cmd/root.go

```diff
@@ -83,26 +166,28 @@ func InitManagerClient() {
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
-	//if cfgFile != "" {
-	//	// Use config file from the flag.
-	//	viper.SetConfigFile(cfgFile)
-	//} else {
-	//	// Find home directory.
-	//	home, err := homedir.Dir()
-	//	if err != nil {
-	//		fmt.Println(err)
-	//		os.Exit(1)
-	//	}
-
-	//	// Search config in home directory with name ".lhctl" (without extension).
-	//	viper.AddConfigPath(home)
-	//	viper.SetConfigName(".lhctl")
-	//}
-	//viper.AutomaticEnv() // read in environment variables that match
 
-	//// If a config file is found, read it in.
-	//if err := viper.ReadInConfig(); err == nil {
-	//	fmt.Println("Using config file:", viper.ConfigFileUsed())
-	//}
+	viper.SetConfigType("yaml")
+
+	if cfgFile != "" {
+		// Use config file from the flag.
+		viper.SetConfigFile(cfgFile)
+	} else {
+		// Find home directory.
+		home, err := homedir.Dir()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		// Search config in home directory with name ".lhctl" (without extension).
+		viper.AddConfigPath(home)
+		viper.SetConfigName(".lhctl")
+	}
+	//viper.AutomaticEnv() // read in environment variables that match
```

Comment:
IMO, unused code should be removed before reaching master - not just commented out.